sdk/cmake/gcc.cmake

```diff
@@ -434,11 +434,11 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug" OR
     set(__spec2def_dbg_arg "--dbg")
```

Comment:
Does this still propagate?
If not, that can be removed now probably?

Reply:
It's still used as default in line 516. I could move this case there though, now.